### DISCUSSION TITLE
Add text to email only at end of all aggregated alerts?

### DISCUSSION THREAD

**Discussion by lymanaukuleles** [jertel/elastalert2#860] [Q&A] [answered]
I currently am sending an HTML formatted email alert with an aggregation period of 10 minutes. I have text in the alert under an HTML table (which is all in alert_text). However is there a way to only send text in the email at the end of the aggregation period? Currently the text is sending with each HTML table. So for example, if I send 3 alerts in the aggregation span of 10 minutes, I will get 3 tables with 3 texts underneath. However I only want 3 tables and 1 text at the very end. 

Is this possible?

Edit: Actually it would be this setting, right ?

`alert_text_type: aggregation_summary_only`

However I want the aggregation summary only for the text, not the tables.

**Comment by nsano-rururu** [comment, marked as answer]
https://github.com/Yelp/elastalert/issues/1722
